collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/pages/Page.java

```diff
@@ -161,4 +168,35 @@ public T checkIfAreaWithKeyIsDisplayed(String key) {
         onData(PreferenceMatchers.withKey(key)).check(matches(isDisplayed()));
         return (T) this;
     }
+
+    public T checkIfStringIsEnabled(int stringID) {
```

Comment:
You don't need this method, you can use checkIfElementWithKeyIsEnabled whcih you added below.